boot/zephyr/boards/stm32n6570_dk_stm32n657xx_fsbl.conf

```diff
@@ -1,3 +1,5 @@
 CONFIG_STM32_MEMMAP=y
 CONFIG_BOOT_MAX_IMG_SECTORS_AUTO=n
 CONFIG_BOOT_MAX_IMG_SECTORS=4096
+
+CONFIG_MULTIPLE_EXECUTABLE_RAM_REGIONS=y
```

Comment:
just FYI: if you want to apply this to **all** boards of a specific soc then you can use a soc Kconfig fragment - check the `socs` folder for examples